py/variable.cpp

```diff
@@ -20,18 +20,32 @@ static PyObject*
 Variable_new( PyTypeObject* type, PyObject* args, PyObject* kwargs )
 {
 	static const char *kwlist[] = { "name", "context", 0 };
-	PyObject* name;
 	PyObject* context = 0;
+
+#if PY_MAJOR_VERSION >= 3
+	const char* name;
```

Comment:
Cool. It doesn't seem like we need the `length` parsing version either. Is the `s|0:__new__` a sufficient parse string?